src/components/views/read/PlayerView.tsx

```diff
@@ -48,9 +48,13 @@ const PlayerView: FC = () => {
       modifyActiveStep(stepIndex);
     }
   };
+  const isSmallScreen = useMediaQuery('(max-width:700px)');
   const stepper = (
     <Stepper nonLinear activeStep={activeStep} alternativeLabel>
-      {steps.map((label, index) => (
+      {(isSmallScreen
+        ? [0, activeStep, steps.length - 1]
+        : steps.map((_, index) => index)
+      ).map((index) => (
```

Comment:
I think it is nice that you want to provide a better layout for small screens. But with this it looks like there are only 3 steps to complete. Maybe later using the [MUI Mobile stepper](https://mui.com/material-ui/react-stepper/#mobile-stepper) would help to have a better display on smaller screen.

Reply:
Also I think it will mess up the clicks. You click review and it just jumps to the next sections ? or it goes to review but that is not what you want ? Not sure about this.

Reply:
Yeh I thought about MUI mobile stepper, and the day I talked with JC, he mentioned a way that whenever the screen is at certain width or less than that, just modify the steps rendering, showing the first, current, and last steps from steps array. And as you said it might be a little messy for the clicks, that's why it would be great to discuss about it a bit.

Reply:
Yes you should change the component that is rendered depending on the screen size (as you are doing with the mediaquery). For me (personal opinion) I think it is strange that the indexes of the steps change depending on screensize.

Reply:
We might open an issue for that, to keep it in mind